docs/housekeeping-abandoned-orgs.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Forks
+permalink: /housekeeping-abandoned-orgs
+---
+
+<div class="chart-placeholder">
+	<h3>Abandoned Organizations</h3>
+	<canvas
+		data-url="{{ site.dataURL }}/organizations-abandoned.tsv"
+		data-type="history"
+	></canvas>
+	<div class="info-box">
+		<p>
+			An organizations is considered <em>abandoned</em> if none of its
```

Comment:
I’d suggest not wrapping these lines manually and leaving this to the editor program. My argument is that diffs tend to become very ugly when flow text is manually rewrapped later on.

Reply:
true. but diffs for very long lines are ugly too. I dunno